### ISSUE TITLE
Map editor : ability to add notes for any entity

### ISSUE THREAD

**Issue by PhoenixII54** [christopho/solarus-quest-editor#281]
(discussed in stream Nr 8 of OLB mapping)
it would be useful to be able to add notes for any active item (chest, switch...) to see what they do refer to for example, or instantely see the content of a chest, without havong to open their editor.

**Comment by christopho** [comment]
Yes, we could just add a "comment" field to entities. It would even be available for regular tiles.
